package.json

```diff
@@ -55,6 +61,7 @@
     "@digitalservice4germany/style-dictionary": "~2.0.0",
     "@eslint/compat": "~1.1.1",
     "@eslint/js": "~9.9.0",
+    "@iconify-json/gravity-ui": "^1.2.1",
```

Comment:
Question: why Gravity UI? I guess this is a design decision but the rest of NeuRIS is using Material Icons. Not sure if we should introduce a separate icon pack for some components, especially since they look quite different...